.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.12.x
+  - 1.13.x
```

Comment:
Why can't 1.12 work with go.mod?

Reply:
More a question than a comment: what is the policy for Go compilers when building OpenShift artifacts?

Reply:
Our CI system has imagestream tags which list supported golang versions. I believe 1.12 is the highest version we currently support (following upstream k8s).

Reply:
@adambkaplan it can work with `1.12`, and since that's the current version in other systems it reverting this change from my PR.